data/production_good_practice.xml

```diff
@@ -109,9 +109,11 @@
                         <varlistentry>
                             <term>/db permissions</term>
                             <listitem>
-                                <warning>The default permissions for /db are 0777, that is to say read-write-update for all!</warning> This should be changed in <emphasis>EVERY</emphasis> production system, to change this to 0774 (rwurwur--) you can run the following XQuery script -  <programlisting language="xquery">declare namespace xmldb = "http://exist-db.org/xquery/xmldb";
-
-xmldb:set-collection-permissions("/db", "admin", "dba", 508)</programlisting>
+                                <para>The default permissions for /db are 0755, which should be sufficient in most cases. In the case you needed to change this, you could do that with (here for 0775):</para>
+                                <programlisting language="xquery">declare namespace xmldb = "http://exist-db.org/xquery/xmldb";
+                                
+                                xmldb:set-collection-permissions("/db", "admin", "dba", 509)</programlisting>
```

Comment:
You should not use `xmldb:set-collection-permissions` instead use `sm:chmod(xs:anyURI("/db"), "rwxr-xr-x")`